File: monWed_sectionA/week11_april6-10/monApril6/Person.java
public class Person {
    protected String name;
    protected int age;

    public Person(String n, int a) {
        name = n;
        age = a;
    }

    public String getType() {
        return "Person";
    }

    @Override
    public String toString() {
        return String.format("%s, age %d", name, age);
    }

    public void birthday() {
        age++;
    }
}

File: monWed_sectionA/week14_april27-may1/week14/Person.cs
public class Person {
    private int _age;

    public string Name {get; init;}


    public int Age
    {
        get{
            return _age;
        }
        set {
            //restrict to 0-120
            if (value >= 0 && value <= 120) {
                _age = value;
            }
            else {
                _age = 0;
            }
        }
    }

    public void Birthday() {
        _age++;
    }

    public void Print() {
        Console.WriteLine("{_name} is {_age} years old", _name, _age);
    }
}